### PR TITLE
Reset passwd validation

### PR DESCRIPTION
**Pull request by ViVaHa**
Adding changes so that the rules for password are shown to be satisfied or not when the user is typing. Only after the user has satisfied all the requirements, he/she can submit
pic1

### CODE REVIEW

**Comment by LinusCastelino** [comment]
Will review code after a quick demo in today's meeting.

**Comment by ViVaHa** [comment]
[UI enhancement for password set component and reset password component](https://trello.com/c/wX6fbCpQ/71-ui-enhancement-for-password-set-component-and-reset-password-component)

**Comment by ViVaHa** [comment]
Updated changes as per those discussed in meeting

File: src/main/react-front-end/src/views/Login/UserAccountComponent.js [hunk 1]
```diff
@@ -139,9 +136,6 @@ export default class UserAccountComponent extends Component {
 			if (currentSaveStatus) {
 				// if the user opted to switch from saving tokens on browser to
 				// storing tokens on the server, we clear all saved tokens in the current browser session.
-
-				// Temporarily commenting out code added in PR 249 for release
-				// Need to handle edge cases arising out of saving Drive token without refresh token
```
Comment (LinusCastelino) [inline src/main/react-front-end/src/views/Login/UserAccountComponent.js:144]: @ViVaHa Please ensure this change is pushed too. Thanks!

**Review by LinusCastelino — APPROVED** [review]
Logic looks good. 
Could not test it manually since an external branch is being merged. Will be tested on master

**Pull request merged**